tool/gravity/cli/status.go

```diff
@@ -170,6 +170,70 @@ func statusOnce(ctx context.Context, operator ops.Operator, operationID string,
 	return status, nil
 }
 
+// statusHistory collects cluster status history and prints out the information
+// to stdout.
+func statusHistory() error {
+	ctx, cancel := context.WithTimeout(context.Background(), defaults.StatusCollectionTimeout)
+	defer cancel()
+	timeline, err := statusapi.Timeline(ctx)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	printEvents(timeline.GetEvents())
+	return nil
+}
+
+func printEvents(events []*pb.TimelineEvent) {
+	w := new(tabwriter.Writer)
+	w.Init(os.Stdout, 0, 8, 1, '\t', 0)
+	if len(events) == 0 {
+		fmt.Fprintln(w, "No status history available to display.")
+		return
+	}
+	for _, event := range events {
+		printEvent(os.Stdout, event)
+	}
+}
+
+func printEvent(w io.Writer, event *pb.TimelineEvent) {
+	// stamp defines default timestamp format.
+	const stamp = "Jan _2 15:04:05 UTC"
```

Comment:
I don't believe we do this consistently throughout gravity, but my preferred date format would be iso 8601.